src/app/users/views/bulkUserActionsEmails.ejs

```diff
@@ -0,0 +1,89 @@
+<a href="../bulk-user-actions" class="link-back">Back</a>
+<div class="row">
+    <div class="col-8">
+        <h1 class="heading-xlarge">
+            Bulk user actions
+        </h1>
+        <p class="lede">Select all the users you want to affect and the action you wish to perform on each one</p>
+    </div>
+</div>
+<div class="row">
+    <div class="col-12">
+        <form method="post" id="form-bulk-user-actions">
+            <input type="hidden" name="_csrf" value="<%= csrfToken %>"/>
+
+            <div class="col-12 %>">
+                <table class="data">
+                    <thead>
+                    <tr>
+                        <th scope="col" class="cwp-15">Name</th>
+                        <th scope="col" class="cwp-35">Email</th>
+                        <th scope="col" class="cwp-25">Organisation</th>
+                        <th scope="col" class="cwp-15">Last Login</th>
+                        <th scope="col" class="cwp-10">Status</th>
+                        <th scope="col" class="cwp-10">Selected</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <% if(locals.users.length === 0) { %>
+                    <tr>
+                        <td colspan="5"><span class="empty-state">No users found</span></td>
+                    </tr>
+                    <% } %>
+                    <% for (let i = 0; i < locals.users.length; i++) { %>
+                    <tr>
+                        <td><%= users[i].name %></td>
+                        <td><span class="breakable"><%= users[i].email %></span></td>
+                        <td>
+                            <% if(users[i].organisation) { %>
+                            <%= users[i].organisation.name %>
+                            <% }else { %>
+                            Unknown
+                            <% } %>
+                        </td>
+                        <td>
+                            <% if(locals.users[i].lastLogin) { %>
+                            <%= locals.moment(locals.users[i].lastLogin).fromNow() %>
+                            <% } else { %>
+                            Never
+                            <% } %>
+                        </td>
+                        <td><%= users[i].status.description %></td>
+                        <td><input type="checkbox" id="user-<%= users[i].id %>" name="user-<%= users[i].id %>" value="<%= users[i].id %>"></td>
```

Comment:
The checkboxes next to each user, we should check with UCD if the smaller checkbox is okay or if they should look like other checkboxes within support, the black-outlined white squares 😄